src/main/java/dk/kea/project/api/StoreController.java

```diff
@@ -1,29 +1,136 @@
 package dk.kea.project.api;
 
 import dk.kea.project.dto.ProductResponse;
+import dk.kea.project.dto.SallingResponse;
 import dk.kea.project.dto.StoreResponse;
+import dk.kea.project.entity.Product;
+import dk.kea.project.entity.Request;
+import dk.kea.project.entity.Store;
+import dk.kea.project.repository.RequestRepository;
+import dk.kea.project.service.ProductService;
 import dk.kea.project.service.SallingService;
+import dk.kea.project.service.StoreService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.time.LocalDateTime;
+import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 @RequestMapping("/api/stores")
 public class StoreController {
-    SallingService sallingService;
-    public StoreController(SallingService sallingService) {
+    @Autowired
```

Comment:
Consider doing constructor dependency injection and remove @Autowired

Reply:
check